doc/reference/trd104-syscalls.md

```diff
@@ -510,13 +510,14 @@ kernel returns `Failure` with an error code of `NODEVICE`.
 
 The Read-Write Allow system call class is how a userspace process
 shares buffer with the kernel that the kernel can read and write. When
-userspace shares a buffer, it can no longer access it. Calling a
+userspace shares a buffer, it should no longer modify it, though the kernel
+will not stop the app from doing so. Calling a
```

Comment:
```suggestion
userspace shares a buffer, it should no longer modify it.
```

I don't think that comment is particularly useful, and if in the future we had a platform where this was possible to properly prevent, it could still be beneficial in kernel code?

Reply:
Yes. The point is that the kernel MAY but is not responsible for doing so.